pyffi/object_models/xml/enum.py

```diff
@@ -84,7 +84,43 @@ def __init__(cls, name, bases, dct):
 
         # set enum values as class attributes
         for item, value in zip(cls._enumkeys, cls._enumvalues):
-            setattr(cls, item, value)
+            setattr(cls, "".join(snakeCase.name_parts(item)), value)
```

Comment:
Missed this on the first run. Eh, not nice, Wonder if we should split out the functionality as it looks to be static rather than class-methody.

Reply:
Sphinx imports and runs the code to document it. With [`nif.xml`](https://github.com/niftools/nifxml/blob/develop/nif.xml#L991) L994 - L1012, the names have spaces in them, with the current code it sets `NifFormat.BSLighingShaderPropertyShaderType.Environment Map` which can't be imported by normal python code, this fixes it and doesn't break any existing code

Reply:
I mean how we could the class generation to the names. They should be static methods.